### PR TITLE
Adding authenticationHelper as export

### PR DESCRIPTION
**Pull request by mbalex99**
adding authenticationHelper as export

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "graphql-tag": "^2.5.0",
     "mocha": "^4.0.1",
     "realm-object-server": "^2.1.1",
-    "realm-object-server-graphql": "file:../realm-object-server-graphql",
+    "realm-object-server-graphql": "file:../realm-graphql-service",
```
Comment (nirinchev) [inline package.json]: Shouldn't this be renamed as well?

File: src/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export * from './graphqlConfig';
 export * from './credentials';
 export * from './user';
+export * from './authenticationHelper';
```
Comment (nirinchev) [inline src/index.ts]: Why is this needed? The authentication helper is internal and shouldn't be used by people directly. Is it to solve this issue: https://github.com/realm/realm-graphql/issues/10?

**Comment by bigfish24** [comment]
@mbalex99 what is this PR solving?
